### DISCUSSION TITLE
Access API for Cost in Inventory

### DISCUSSION THREAD

**Discussion by katkosc** [Sellbrite/Sellbrite-API#50] [Q&A]
Hi guys,

is it possible to access the "cost" of a product in the inventory? If yes, could you give us a little guideline?
My developer says, he cant access it.

Thanks,
Kat
![Screenshot 2024-09-17 at 3 05 14 PM](https://github.com/user-attachments/assets/b3ba003b-19c2-43f6-bfa3-c3da9d63ee4e)

**Comment by rengler-godaddy** [comment]
Hello - Thanks for reaching out! 

You should be able to retrieve that information using the inventory endpoint. More information can be found on the following link: https://developer.sellbrite.com/endpoints/inventory 

If you’re seeing an issue pulling that information let us know.

**Reply by rwrllc** [reply]
How do we update the cost though, the PUT seems to fail
